ui/src/Common/Header/Header.test.tsx

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright (c) 2022 Ford Motor Company
+ * All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import {screen} from '@testing-library/react';
+import Header from './Header';
+import {RunConfig} from 'Types/RunConfig';
+import flagsmith from 'flagsmith';
+import {renderWithRecoil} from 'Utils/TestUtils';
+import {MemoryRouter} from 'react-router-dom';
+import {
+    enableInviteUsersToSpace,
+    openAccountDropdown,
+    shouldHideHeaderAccountDropdown,
+    shouldOnlyShowSignoutButtonInAccountDropdown,
+    shouldRenderLogoAsDashboardLinkInHeader,
+    shouldRenderStaticLogo,
+    shouldShowAllAccountDropdownOptions,
+} from 'Utils/HeaderTestUtils';
+import {MutableSnapshot} from 'recoil';
+import {CurrentUserState} from 'State/CurrentUserState';
+
+const debounceTimeToWait = 100;
+
+describe('Header', () => {
+    beforeEach(() => {
+        window.runConfig = {invite_users_to_space_enabled: true} as RunConfig;
```

Comment:
can we utilize that enableUsersToSpace function from utils here?

Reply:
Yes! Good catch